internal/wasm/binary/element_test.go

```diff
@@ -0,0 +1,255 @@
+package binary
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/tetratelabs/wazero/internal/testing/require"
+	"github.com/tetratelabs/wazero/internal/wasm"
+)
+
+func asNUllableIndex(v uint32) wasm.NullableIndex {
```

Comment:
maybe `uint32Ptr` to save wrapping? "nullable index" isn't terribly necessary in tests

Reply:
works for me